demos/asp-net-core/ITOps.ViewModelComposition/Subscription.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using System;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition
+{
+    class Subscription<T> : ISubscription
+    {
+        private Func<dynamic, T, RouteData, IQueryCollection, Task> subscription;
```

Comment:
![image](https://user-images.githubusercontent.com/677704/31090424-dc12ea98-a7a7-11e7-92f5-a9e11e7f36b1.png)

@mauroservienti did you intend to rename this?

Personally I like using a verb for the name of a delegate, since then invocation is of the form `handle()`, but I won't oppose `handler` if that's what you prefer.

Reply:
done